### DISCUSSION TITLE
disable HTTPS for Kibana, using reverse proxy for SSL for frontend

### DISCUSSION THREAD

**Discussion by benisai** [sherifabdlnaby/elastdocker#50] [Questions & Troubleshooting] [answered]
Hello, great work thus far. How can SSL be disabled in the UI? I'm using a reverse proxy and it will do SSL for me.

**Comment by sherifabdlnaby** [comment, marked as answer]
You can disable it.
https://github.com/sherifabdlnaby/elastdocker/blob/8485d92a03267bf43cfa769facc2cd2745e2e3ba/kibana/config/kibana.yml#L12

**Reply by benisai** [reply]
@sherifabdlnaby  that worked great. -- Any way to disable all SSL for all products (elastic, logstash and kibana)? I'm using this for internal testing in my homelab and dont want to mess with SSL certs on hosts. Any way you can provide some assistance/knowledge?

**Reply by sherifabdlnaby** [reply]
You'll find a similar configuration in Elasticsearch, but when you disable it make sure to change connection strings from `https://` to `http://`... Think about it tho, as now many of Elastic Stack features need SSL sertup as a prerequiste.

**Reply by benisai** [reply]
Ah yes. I figured it out after posting, but I reverted it, too many errors in the ui. Question tho, Can you incorporate fleet-server into your scripts? Seems like that's the go forward for elastic team. Use fleet server for everything. 
I tried setting up fleet server in a docker container but it just won't connect in kibana or elastic.
